### PR TITLE
[RESOURCES] updated FAQ section under resources

### PR DESCRIPTION
**Pull request by asubedy**
Signed-off-by: asubedy <frexpe@pm.me>

**Description**
Added collection of Faqs.

This PR fixes #

**Notes for Reviewers**
Advice needed on the images I have used for internship and service mesh thumbnail.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by asubedy** [comment]
Ok I think the build is failing because I didn't include the `tags` field on the files. Will make this change.

**Comment by l5io** [comment]
🚀 Preview for commit ac82730aa15f76bffe40227c4959edd4fcad2fba at: https://61fcdb9cb310b45af286074a--layer5ng.netlify.app

**Comment by asubedy** [comment]
I don't know why the background images are not visible. It was visible in my local system. 🤔. Can we not use variables on `styled.div`?

**Comment by debo19** [comment]
> I don't know why the background images are not visible. It was visible in my local system. 🤔. Can we not use variables on `styled.div`?

@asubedy I think you're commenting on the wrong PR 😆 Yeah don't use variable there since we are not using `<StaticImages>` in those files.

**Comment by asubedy** [comment]
> > I don't know why the background images are not visible. It was visible in my local system. 🤔. Can we not use variables on `styled.div`?
> 
> @asubedy I think you're commenting on the wrong PR 😆 Yeah don't use variable there since we are not using `<StaticImages>` in those files.

wow 😆  yep understood 😬

File: src/collections/resources/faq/internship-faq.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Internship FAQs"
+thumbnail: ../../../assets/images/workshops/workshops.png
+category: FAQ
+tags:
+  - Internship 
+  - FAQ
+type: FAQ
+product: FAQ
```
Comment (debo19) [inline src/collections/resources/faq/internship-faq.mdx:9]:
```suggestion
```
FAQ is not a product, we can skip this.

File: src/collections/resources/faq/layer5-faq.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Layer5 FAQs"
+thumbnail: ../../../assets/images/layer5/layer5-tagline/png/layer5-tag-dark-bg.png
+category: FAQ
+type: FAQ
+tags: 
+  - Layer5
+  - FAQ
+product: FAQ
```
Comment (debo19) [inline src/collections/resources/faq/layer5-faq.mdx:9]:
```suggestion
```

File: src/collections/resources/faq/meshery-faq.mdx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Meshery FAQs"
+thumbnail: ../../../assets/images/meshery/full-logo/meshery-logo-tag-dark-text-side.png
+category: FAQ
+tags: 
+  - Meshery
+  - FAQ
+type: FAQ
+product: FAQ
```
Comment (debo19) [inline src/collections/resources/faq/meshery-faq.mdx:9]:
```suggestion
product: Meshery
```

File: src/collections/resources/faq/service-mesh-faq.mdx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Service Mesh FAQs"
+thumbnail: ../../../assets/images/service-mesh-icons/linkerd/horizontal/color/linkerd-horizontal-color.png
```
Comment (debo19) [inline src/collections/resources/faq/service-mesh-faq.mdx:3]: Using just the Linkerd image is not appropraite. You can just use the `../service-mesh.svg` for now

File: src/collections/resources/faq/service-mesh-faq.mdx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+category: FAQ
+tags: 
+  - Service Mesh
+  - FAQ
+type: FAQ
+product: FAQ
```
Comment (debo19) [inline src/collections/resources/faq/service-mesh-faq.mdx:9]:
```suggestion
```

**Review by debo19 — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 475a963e87968f29ab19ef5c9902d6ec2f15c4b9 at: https://61fd25b6ea4966097dd41a56--layer5ng.netlify.app

File: src/collections/resources/faq/meshery-faq.mdx (hunk 3 above)
Comment (debo19) [reply src/collections/resources/faq/meshery-faq.mdx]: @asubedy just this product can be "Meshery". Rest looks good 🚀

Comment (asubedy) [reply src/collections/resources/faq/meshery-faq.mdx]: Oh ok. 👍

**Comment by l5io** [comment]
🚀 Preview for commit 1e853b2c9790f24858e4e912b92a48f9bb2f09a4 at: https://61fd32e24343480d2faa1364--layer5ng.netlify.app

File: src/collections/resources/faq/meshery-faq.mdx [hunk 6]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Meshery FAQs"
+thumbnail: ../../../assets/images/meshery/full-logo/meshery-logo-tag-dark-text-side.png
+category: FAQ
+tags: 
+  - Meshery
+  - FAQ
+type: FAQ
+produc: Meshery
```
Comment (debo19) [inline src/collections/resources/faq/meshery-faq.mdx:9]:
```suggestion
product: Meshery
```

**Review by debo19 — CHANGES_REQUESTED** [review]

**Review by debo19 — APPROVED** [review]
LGTM 🎉  Thanks @asubedy

**Pull request merged**